### PR TITLE
Fix Pedestal Issue #610

### PR DESCRIPTION
**Pull request by bonkydog**
https://github.com/pedestal/pedestal/issues/610

`url-for` throws and exception when a context path is supplied and used for a route with a path parameter in the first segment.

The logic appeared to assume that path segments are always strings and was using `empty?` to check for the empty string.  `empty?` expects a sequence.  A keyword is not a sequence, so empty was throwing an exception.

### CODE REVIEW

File: route/src/io/pedestal/http/route.clj [hunk 1]
```diff
@@ -249,7 +249,7 @@
                                   :context-path-parts context-path-parts)
                        ;;(concat context-path-parts path-parts)
```
Comment (hlship) [inline route/src/io/pedestal/http/route.clj:250]: This comment should be deleted.

**Review by hlship — APPROVED** [review]
Maybe delete that comment before merging.

File: route/src/io/pedestal/http/route.clj (hunk 1 above)
Comment (bonkydog) [reply route/src/io/pedestal/http/route.clj:250]: ah, thanks!

**Comment by hlship** [comment]
Above changes accidental, this was fixed in 0.6.0.

**Pull request merged**